ci/pipelines/meta/deploy_meta.sh

```diff
@@ -105,25 +118,12 @@ if [[ "$(tail -n1 flyOutput.log)" == "bailing out" ]]; then
   exit 1
 fi
 
-if [[ "${GEODE_FORK}" != "${UPSTREAM_FORK}" ]]; then
```

Comment:
If everything from here down is just improving the post-deploy watching, that's fine.